config/samples/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+## Append samples you want in your CSV to this file as resources ##
```

Comment:
We are maintaining kustomize files for samples within `config/hub/samples` and `config/dr-cluster/samples`. This one will go to the hub samples  as it is hub specific and is already present there, so comment is really to remove this file from the PR :)

Reply:
removed!